### DISCUSSION TITLE
Moved account, can I delete old one

### DISCUSSION THREAD

**Discussion by hemitheo** [mastodon/mastodon#20740] [Q&A] [answered]
I moved my account to a different server. Kept the same username and also my followers. Can I now delete the old account or would this affect operation of the new one?

**Comment by madeofstown** [comment, marked as answer]
You can try redirecting the old domain to the new one for a few weeks and then it's probably safe to totally remove it after a while. You should be able to check logs to see if people are connecting with the old domain.

**Reply by hemitheo** [reply]
Thank you! Will do. :-)
